### DISCUSSION TITLE
INTERFACE CHOICE

### DISCUSSION THREAD

**Discussion by Z-YHWH-24** [Z-YHWH-24/ENEO-AI-PROJET#2] [Ideas]
**SALUT , CETTE DISCUSSION EST UNIQUEMENT FAITES POUR LE CHOIX DES L'INTERFACE A DEVELOPPER

COMMENT SIGNALER SON CHOIX  ?

NOM DE L'INTERFACE CHOISIS ET UN PHOTO**

**Comment by audreyossalobo** [comment]
je choisi le 2 interface des formulaires

**Comment by Z-YHWH-24** [comment]
EXAMPLE  

L'INTERFACE DE BIENVENUE

![Screenshot from 2024-07-28 21-24-07](https://github.com/user-attachments/assets/674db3ee-df1a-496d-b10a-9c9d2cb0e427)

**Reply by Z-YHWH-24** [reply]
C'EST JUSTE UN EXAMPLE VOUS DEVEZ AJOUTER LA  PHOTO DU RESULTATS FINALE
